core/schema/git.graphqls

```diff
@@ -1,24 +1,57 @@
 extend type Query {
-    "Queries a git repository."
-    git(url: String!, keepGitDir: Boolean): GitRepository!
+    """
+    Queries a git repository.
+    """
+    git(
+        """
+        Url of the git repository.
+        Can be formatted as https://{host}/{owner}/{repo}, git@{host}/{owner}/{repo}
+        Suffix ".git" is optional.
```

Comment:
I don't think documenting git is not our problem. I would remove this line.

Reply:
I think it's still useful for users to understand what kind of input they can provide.

Reply:
Sorry, I was just talking about the last line (L9) about the `.git` suffix :)
Of course the string format expected is very useful and important here.